homeassistant/components/flux/switch.py

```diff
@@ -50,6 +50,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_UNIQUE_ID = "unique_id"
```

Comment:
Isn't this something that can/is stored in a upperlying const.py? It seems very generic.

Reply:
Yeah, you'd expect; unfortunately, that lead to different issue in this specific case.
Eventually, this integration needs a revamp as a whole (if we even want that to happen, I'm not sure).

../Frenck

Reply:
I see. Then I can't find more points now to nitpick on. ;)